spec/models/course_enrichment_spec.rb

```diff
@@ -210,9 +210,9 @@
 
     context 'fee based course' do
       it { should validate_presence_of(:fee_uk_eu).on(:publish) }
-      # it { should validate_presence_of(:about_course).on(:publish) }
+      it { should validate_presence_of(:about_course).on(:publish) }
```

Comment:
@misaka were these commented out because they should be removed? I uncommented them because they pass, but they seem to duplicate part of the above tests.

Reply:
Ya, I commented them out just because they duplicate. Wasn't sure about their value but am happy to leave them if you think they have value.

Reply:
No, let's remove them then.